src/main/java/com/spartronics4915/frc2020/Constants.java

```diff
@@ -1,4 +1,8 @@
 package com.spartronics4915.frc2020;
 
 public final class Constants {
+    public static final int kFlywheelMasterID = -1;
```

Comment:
Be sure to add all of these devices and IDs to the [2020 Control Map](https://docs.google.com/spreadsheets/d/14O3Dj1Bl6tqiHFrCJ7wyVRDOqS5EZ11p4q9j2rYS-3Q/edit#gid=0).